File: 项目开发进度2009年2月9日/SeeMapCell/MapXtreme/6.6/Samples/VisualStudio2005/Web/Features/CustomTools/CustomToolsCS/App_Code/CustomizedWebTools/CustomizedWebTools.cs
using System;
using MapInfo.WebControls;

namespace CustomizedWebTools
{
	/// <summary>
	/// PinPointWebTool control for AddPinPointCommand and ClearPinPointCommand tools.
	/// </summary>
	public class PinPointWebTool : MapInfo.WebControls.WebTool
	{
		public PinPointWebTool()
		{
			//Command = "";
			this.ClientInteraction = ClientInteractionEnum.ClickInteraction.ToString();
			Active = false;
			//CursorImageUrl = string.Format("{0}/MapInfoWeb{1}.cur", MapInfo.WebControls.Resources._resourceFolderMain, Command);
			CursorImageUrl = "";
		}
	}
}

File: 项目开发进度2009年2月9日/SeeMapCell/MapXtreme/6.6/Samples/VisualStudio2003/Web/Features/Tools/vb/CustomizedWebTools/CustomizedWebTools.vb
Imports System.Web
' <summary>
' PinPointWebTool control for AddPinPointCommand and ClearPinPointCommand tools.
' </summary>
Public Class PinPointWebTool
    Inherits MapInfo.WebControls.WebTool
    Public Sub New()
        ' Command = ""
        Me.ClientInteraction = ClientInteractionEnum.ClickInteraction.ToString()
        Active = False
        CursorImageUrl = ""
    End Sub
End Class
